### DISCUSSION TITLE
paddle2onnx转rec_server_infer为onnx后，推理速度非常慢

### DISCUSSION THREAD

**Discussion by wenQ0313** [PaddlePaddle/PaddleOCR#13448] [Q&A]
一、运行环境：
paddlepaddle-gpu:2.5.0
paddle2onnx: 1.1.0
onnx: 1.14.1
onnxruntime-gpu: 1.16.3

二、转onnx命令
![1721611506260](https://github.com/user-attachments/assets/082a02c3-5286-4bc2-b024-0d1c7e460db8)

三、推理速度很慢的代码位置：推理时长是不使用onnx的时长的两三倍
![image](https://github.com/user-attachments/assets/490da0dc-c4af-42f7-9775-ddd960836049)

**Comment by SWHL** [comment]
onnxruntime-gpu推理的确会很慢，因为ort对动态输入的支持很差，不建议用ort-gpu推理onnx模型
详细参见我之前写的博客：[docs](https://rapidai.github.io/RapidOCRDocs/blog/2022/10/04/which-inference/)

**Comment by wenQ0313** [comment]
![image](https://github.com/user-attachments/assets/9c837119-c1bc-4efd-a896-a3e3cdc5da0d)
你好，使用你推荐的方式，我直接使用源码中的python/rapdiocr_paddle/main.py 尝试使用v4_server_infer模型做推理，但是检测模型的结果为空，请问是什么原因呢
